test/e2e/pages/vl-map.page.js

```diff
@@ -38,15 +38,14 @@ class VlMapPage extends Page {
   async _getMap(selector) {
     const map = await new VlMap(this.driver, selector);
     await this.driver.wait(async () => {
-      let resolved = false;
-      const promise = this.driver.executeScript('return arguments[0].ready', map);
       try {
-        await this.driver.wait(async () => await promise.then(() => resolved = true), 1000);
-        return resolved;
+        await this.driver.wait(this.driver.executeScript('return arguments[0].ready', map), 1000);
+        return true;
       } catch (error) {
         return false;
       }
     });
```

Comment:
zouden we dat dan niet beter verhuizen naar VlMap of zo? dan zit het ook niet gedupliceerd ...

Reply:
De code waar je naar verwijst komt uit het e2e page object, niet component page object. Die zou ik bewust niet willen mengen. Verhuizen naar VlMap en via page object en component page object ernaar verwijzen is vermoedelijk wel mogelijk en zo voorkomen we duplicatie.

Reply:
Zoiets ja